webapp/channels/src/components/admin_console/multiselect_settings.tsx

```diff
@@ -42,18 +54,16 @@ export default class MultiSelectSetting extends React.PureComponent {
     };
 
     calculateValue = () => {
-        return this.props.selected.reduce((values, item) => {
-            const found = this.props.values.find((e) => {
-                return e.value === item;
-            });
-            if (found !== null) {
+        return this.props.selected.reduce<Option[]>((values, item) => {
+            const found = this.props.values.find((e) => e.value === item) as Option | undefined;
```

Comment:
```suggestion
            const found = this.props.values.find((e) => e.value === item);
```
We don't have to explicitly type this as it will be inferred from the type of `values` above